pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -1520,6 +1523,46 @@ var _ = Describe("Converter", func() {
 			Expect(domainSpec.Devices.Rng).ToNot(BeNil())
 		})
 
+		table.DescribeTable("Validate that QEMU SeaBios debug logs are ",
+			func(toDefineVerbosityEnvVariable bool, virtLauncherLogVerbosity int, shouldEnableDebugLogs bool) {
```

Comment:
I figure you wanted to implement using `toDefineVerbosityEnvVariable` but forgot?

Reply:
You're absolutely right.. Forgot about it, I'll add it and re-upload.

Regarding go compiler not complaining, I found [this answer](https://stackoverflow.com/questions/22549228/why-does-go-allow-compilation-of-unused-function-parameters) on stack overflow. It says:

> There's no official reason, but the reason given on golang-nuts is:
> 
> Unused variables are always a programming error, whereas it is common to write a function that doesn't use all of its arguments.
> One could leave those arguments unnamed (using _), but then that might confuse with functions like
> func foo(_ string, _ int) // what's this supposed to do?
> The names, even if they're unused, provide important documentation.
> Andrew

Not very convincing if you ask me. In golang-nuts thread they speak about maybe slightly changing it in the future.

Anyways, thanks for noticing it !

Reply:
@dhiller In second thought, I think it's necessary in a situations when a type is needed to fit to a certain interface. It is reasonable that the interface's function has 2 parameters but the implementation for a specific type only uses one of them.

They said in golang-nuts thread that maybe in the future the compiler will allow this only when the unused parameter is unbounded to a variable ( e.g. func(x string, _ int) ) but will fail compilation if a bounded variable isn't used.

Reply:
Sure, for interfaces that may make sense. Thanks for researching this, valuable information :+1: